include/entity/scene.h

```diff
@@ -0,0 +1,27 @@
+#ifndef ENTITY_SCENE_H
+#define ENTITY_SCENE_H
+
+#include <vector>
+#include <entity/light.h>
+#include <graphic/color.h>
+
+class Scene {
+private:
+    std::vector<Light> lights;
+    Color environmentColor; // QUESTION: Should this be a Color or a Pixel?
```

Comment:
O que tu achas @kinhosz? Vamos seguir com `Color` ou `Pixel` aqui?

Reply:
Color. Minha justificativa é a seguinte:

Todas as cores passam pela fórmula de phong, envolvendo multiplicacoes, somas e etc... Trabalhar com o não normalizado vai dar mais trabalho.

Color onde tiver cor.
Pixel quando for printar na tela. Logo, o único lugar no projeto que faz sentido ter pixel até agora é no frame, o resto deveria ser color mesmo.

Reply:
De acordo, vou manter como color então.